command/sub8_missions/missions/align_channel.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import division
```

Comment:
Without the search pattern I think it would be a lot more condensed. But oh well, bigger is better